app/Http/Controllers/Admin/MeetingsController.php

```diff
@@ -41,16 +40,16 @@ public function update(Request $request, $id)
             'category_id'   => 'required',
             'level_id'      => 'required',
             'date'          => 'required|date|after_or_equal:today',
-            'start_at'      => 'required'
+            'start_at'      => 'required|integer|min:0|max:24'
         ]);
         $meeting = Meeting::withTrashed()->findOrFail($id);
         $meeting->title         = $request->title;
         $meeting->room_id       = $request->room_id;
         $meeting->category_id   = $request->category_id;
         $meeting->level_id      = $request->level_id;
         $meeting->date          = $request->date;
-        $meeting->start_at      = $request->start_at;
-        $meeting->status_id     = self::DEFAULT_STATUS_ID;
+        $meeting->start_at      = date('G:i:s', strtotime($request->start_at . ':00:00'));
```

Comment:
Do you mean H:i:s ? Please correct

Reply:
Ahh okay, got it you want no trailing '0' format

Reply:
Yes I want to display time like 6:00

Reply:
I changed it into H